cpp/example/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 include(${CMAKE_SOURCE_DIR}/cmake/UnitTesting.cmake)
 
+if(ENABLE_DEAL_II)
```

Comment:
Can you indent the lines below. It makes it easier to see what's in the if.

Reply:
I expect clang-format will do that.

Reply:
Oh right this is CMake...  I will do that.